android/build.gradle

```diff
@@ -11,8 +11,8 @@ buildscript {
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion 25
+    buildToolsVersion "25.0.0"
```

Comment:
hmm u sure?

```
import android.os.LocaleList;
                 ^
  symbol:   class LocaleList
  location: package android.os
/cook/client/node_modules/react-native-i18n/android/src/main/java/com/AlexanderZaytsev/RNI18n/RNI18nModule.java:47: error: cannot find symbol
    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
```

I got similar error when using compileSdkVersion 23.
And I checked it's API level 24 functions
https://developer.android.com/reference/android/os/LocaleList.html